uploader/src/main/resources/cfn-template.yaml

```diff
@@ -39,7 +36,7 @@ Parameters:
   AlertWebhook:
     Description: "Where CloudWatch alerts are sent"
     Type: "String"
-    Default: ""
+    # Optional in DEV/CODE (just fill in with a dummy string)
```

Comment:
I'd remove the comment and add `dummy-string` as the parameter for DEV and CODE.